### PR TITLE
clang-format + scripts

### PR DESCRIPTION
**Pull request by adriandavila**
As far a clang-format goes, pretty much just the default LLVM config atm.

Of note, `ortoa-lib.sh` which is going to become a command/script library for the project. Started it off with formatters, can add linters, build commands, cleanup commands, etc.

### CODE REVIEW

File: ortoa-lib.sh
Comment (jasondu7297) [inline ortoa-lib.sh:1]: (nit) Can `ortoa-lib.sh` be moved in /scripts?

File: scripts/clang-format-all.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    fi
+    if [ "$1" = "/" ]; then
+        return 1
+    fi
+    find-dominating-file "$(realpath "$1"/..)" "$2"
+    return $?
+}
+
+# Run clang-format -i on all of the things
+for dir in "$@"; do
```
Comment (jasondu7297) [inline scripts/clang-format-all.sh:61]: Will the /build directory be included in this? I think we should avoid touching any build files

Comment (adriandavila) [reply scripts/clang-format-all.sh:61]: Gets invoked like this from the command library
`source scripts/clang-format-all.sh host/ enclave/ crypto/ client/`

\therefore not an issue as long as you don't play with invoking this bash script directly.

File: ortoa-lib.sh
Comment (adriandavila) [reply ortoa-lib.sh:1]: sure, done

**Pull request merged**